Dockerfile

```diff
@@ -141,7 +141,7 @@ RUN git submodule update --init --recursive
 ARG FIN_DIR=$MIOPEN_DIR/fin
 WORKDIR $FIN_DIR
 # Can be a branch or a SHA
-ARG FIN_BRANCH=
```

Comment:
Please open a PR for this branch in MIFin. We'll keep the branch here on fin's develop.

Reply:
https://github.com/ROCmSoftwarePlatform/MIFin/pull/87